jest.config.js

```diff
@@ -17,5 +17,8 @@ module.exports = {
   coveragePathIgnorePatterns: [
     '/node_modules/',
     '<rootDir>/packages/babel-plugin/test/util.js'
-  ]
+  ],
+  snapshotFormat: {
+    escapeString: false
```

Comment:
While we are at it - could you update all snapshots with this? Unless you have a preference *not* to use this option?
```suggestion
    escapeString: false,
    printBasicPrototype: false
```

Reply:
Nah, that config will be default in jest 29 probably 🙂